webapp/src/desktop_notifications.ts

```diff
@@ -0,0 +1,26 @@
+import {Channel} from '@mattermost/types/channels';
+import {Post} from '@mattermost/types/posts';
+
+import {DesktopNotificationArgs, Store} from 'src/types/mattermost-webapp';
+
+import {ringingEnabled} from 'src/selectors';
+import {isDmGmChannel} from 'src/utils';
+
+export function desktopNotificationHandler(store: Store, post: Post, channel: Channel, args: DesktopNotificationArgs): {error?: string, args?: DesktopNotificationArgs} {
+    if (args.notify) {
+        // Calls will notify if:
+        //  1. it's a custom_calls post (call has started)
+        //  2. in a DM or GM channel
+        //  3. calls is enabled and is v0.17.0+ (it is if this is running)
```

Comment:
Would the version be `>= v0.18.0`? At least for the new notification. Also I suppose there's the extra requirement of `MM >= v8.1`

Reply:
Ah, right, thanks, fixed!